ext/spl/tests/bug60161.phpt

```diff
@@ -0,0 +1,39 @@
+--TEST--
+Bug #60161 (Implementing an interface extending Traversable is order dependent)
+--CREDIT--
+KCPHPUG TestFest 2017 - Eric Poe
+--FILE--
+<?php
+interface Foo extends \Traversable {}
+interface Bar {}
+
+Class A implements \IteratorAggregate, Foo, Bar
+{
+  public function getIterator()
+  {
+    return new \ArrayIterator([]);
+  }
+}
+
+class B implements Foo, \IteratorAggregate, Bar
+{
+  public function getIterator()
+  {
+    return new \ArrayIterator([]);
+  }
+}
+
+class C implements Foo, Bar, \IteratorAggregate
+{
+  public function getIterator()
+  {
+    return new \ArrayIterator([]);
+  }
+}
+
+echo "OK";
+?>
+--XFAIL--
```

Comment:
We won't accept `XFAIL` tests :confused:

Reply:
@carusogabriel How do you recommend that we verify that a bug still exists?

Reply:
> How do you recommend that we verify that a bug still exists?
@ericpoe http://bugs.php.net.

Just to explain why we won't accept this kind of tests: is better to track an existing bug via de bugtracker, rather than have a failing test in our source code.